index.html

```diff
@@ -462,7 +462,7 @@ <h1>Students</h1>
     </div>
     <!-- end of row -->
     <div class="row equal-row">
-      {% for student in site.data.students limit:30 %}
+      {% for student in site.data.students limit:100 %}
```

Comment:
Don't limit students let's show them all. Remove the limit.

Reply:
I removed the limit just now.